netci.groovy

```diff
@@ -2199,6 +2359,51 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                     // Basic archiving of the build, no pal tests
                     Utilities.addArchival(newJob, "bin/Product/**,bin/obj/*/tests/**/*.dylib,bin/obj/*/tests/**/*.so", "bin/Product/**/.nuget/**")
                     break
+                case 'arm':
+                    // Non-Windows ARM cross builds on hardware run on Ubuntu only
+                    assert (os == 'Ubuntu')
+
+                    // Debugging:
+                    // 1. show the IP address of the machine we're running on.
```

Comment:
I would strongly recommend to not publicly publish ip numbers.

Reply:
I would suggest changing the machine hostnames to unique names and printing the hostname instead.